yt_dlp/extractor/globo.py

```diff
@@ -137,8 +137,8 @@ def _real_extract(self, url):
             formats, subtitles = self._extract_m3u8_formats_and_subtitles(
                 main_source['url'], video_id, 'mp4', m3u8_id='hls')
         self._merge_subtitles(traverse_obj(main_source, ('text', ..., {
-            'url': ('subtitle', 'srt', 'url', {url_or_none}),
-        }, all, {subs_list_to_dict(lang='en')})), target=subtitles)
+            'url': ('caption', 'srt', 'url', {url_or_none}),
```

Comment:
unfamiliar with the project – should I update `_TESTS` above to cover this?

Reply:
you could add `'subtitles': 'count:1'` (or however many) to one of the impacted test cases

and to go with my above suggestion:
```suggestion
            'url': ('srt', 'url', {url_or_none}),
```

Reply:
for sure, thank you, updated per suggestion!

added the test but not sure it's asserting anything, it passes with `'subtitles': 'count:42'` too 🤔 

I imagine it might be due to that entry requiring a subscription, so it's not running by default?

Reply:
what command are you using to run the test? have you set up `test/local_parameters.json` so that you can authenticate when running tests?

Reply:
running tests with just `hatch test GloboIE`

is there a flag to make it print logs/debug? not seeing with `hatch test --help`
sorry, total noob with this tooling!

this Globo extractor doesn't seem to do well with user and pass, I always get a 401 despite supplying good creds. but it works with `--cookies-from-browser chrome`, so I was trying to configure `local_parameters.json` like:

```
{ "cookiesfrombrowser": ["chrome"] }
```

but that doesn't seem to change things, and the test still passes despite `'subtitles': 'count:42'`

Reply:
`hatch test GloboIE` only runs the first extractor test case. you need to use `hatch test GloboIE_all` or `hatch test GloboIE_2`

Reply:
oh man, works now! some tests were outdated, just updated them all to match latest data, all passing consistently here with the `local_parameters.json` above